src/features/movieListSlice.js

```diff
@@ -22,6 +23,9 @@ const movieListSlice = createSlice({
         fetchDataError: (state) => {
             state.status = "error";
         },
+        setQuery: (state, { payload }) => {
+            state.query = payload.query;
+        },
         goToPage: (state, { payload }) => {
```

Comment:
Remember to add here 
`state.status="loading"; `

Reply:
added :)